### PR TITLE
Add Minecraft server files to gitignore

### PR DESCRIPTION
**Pull request by rmsy**
These files are generated when you run the embedded Minecraft server in the project directory.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,17 @@ Commons/core/src/main/i18n/translations/*
...
+server.properties
+sportpaper.yml
+update
+usercache.json
+whitelist.json
+logs/
+plugins/
+world/
+world_nether/
+world_the_end/
```
Comment (TheMolkaPL) [inline .gitignore]: Aren't these inside the `matches/` directory?

**Comment by rmsy** [comment]
For matches, maybe (it’s been a while) - but﻿ I believe these are the directories that are generated when the server starts initially.

File: .gitignore (hunk 1 above)
Comment (rmsy) [reply .gitignore]: For matches, maybe (it’s been a while) - but﻿ I believe these are the directories that are generated when the server starts initially

**Comment by Pablete1234** [comment]
Should just have a server folder added to the gitignore, and change the working directory to there

**Pull request merged**